File: src/main/java/sample/SampleController.java
package sample;

import javafx.event.ActionEvent;
import javafx.scene.control.Alert;
import javafx.scene.control.Label;

public class SampleController {
  public Label helloWorld;

  public void sayHelloWorld(final ActionEvent actionEvent) {
    helloWorld.setText("Hello World!");
    Alert alert = new Alert(Alert.AlertType.INFORMATION, "Hello World!?");
    alert.showAndWait();
  }
}

File: src/main/java/xarmanta/application/SampleController.kt
package xarmanta.application

import javafx.event.ActionEvent
import javafx.scene.control.Alert
import javafx.scene.control.TextArea

class SampleController {
    lateinit var console: TextArea

    fun sayHelloWorld(actionEvent: ActionEvent?) {
        console.appendText("zurixe\n")
        val alert = Alert(Alert.AlertType.INFORMATION, "Hello World!?")
        alert.showAndWait()
    }
}
